src/main/java/dao/impl/BasketDaoJdbcImpl.java

```diff
@@ -19,8 +18,8 @@
 
     private static final Logger logger = Logger.getLogger(BasketDaoJdbcImpl.class);
     private static final String ADD_PRODUCT = "INSERT INTO basket(product_id, product_name, " +
-                    "product_description, product_price, user_id) " +
-                    "VALUES('%s', '%s', '%s', '%s', '%s')";
+            "product_description, product_price, user_id) " +
```

Comment:
we need to use prepare statement in all queries

Reply:
I don't understand what you mean. What I must fix in this line?

Reply:
prepared statement look like this:
private static final String ADD_PRODUCT = "INSERT INTO basket(product_id, product_name, " +
"product_description, product_price, user_id) VALUES(?, ?, ?, ?, ?)";

Reply:
oh, I forgot for this solution.